pkg/fileutils/fileutils_test.go

```diff
@@ -508,7 +509,7 @@ var matchTests = []matchTest{
 	{"*c", "abc", true, nil},
 	{"a*", "a", true, nil},
 	{"a*", "abc", true, nil},
-	{"a*", "ab/c", false, nil},
+	{"a*", "ab/c", true, nil},
```

Comment:
See https://github.com/docker/docker/pull/31669#discussion_r105052639 . The public method runs on a parent as well that is in the same level as the pattern. Previous version tested the behavior of an internal function that was not used anywhere. I could make the test us `Pattern` private method directly but that wouldn't test anything useful.